projects/1.19/assets/enders-delight/endersdelight/lang/zh_cn.json

```
@@ -0,0 +1,34 @@
+{
+  "block.endersdelight.chorus_crate": "箱装紫颂果",
+  "block.endersdelight.chorus_pie": "紫颂果派",
+  "block.endersdelight.endstone_stove": "末地石炉",
+  "block.endersdelight.stuffed_shulker_block": "填馅潜影贝",
+  "death.attack.wasWatered": "%1$s被浇了水",
+  "effect.endersdelight.phasing": "末影折跃",
+  "endersdelight.block.stuffed_shulker.use_shulker_bowl": "你需要一个碗才能吃这个",
+  "item.endersdelight.chorus_juice": "紫颂果汁",
+  "item.endersdelight.chorus_pie_slice": "紫颂果派切片",
+  "item.endersdelight.chorus_stew": "紫颂炖菜",
+  "item.endersdelight.chorus_stew_wood": "紫颂炖菜",
+  "item.endersdelight.crawling_sandwich": "蠕动三明治",
+  "item.endersdelight.crispy_skewer": "酥脆串",
+  "item.endersdelight.ender_paella": "末影海鲜饭",
+  "item.endersdelight.ender_paella_wood": "末影海鲜饭",
+  "item.endersdelight.ender_shard": "末影碎片",
+  "item.endersdelight.enderman_sight": "末影人眼球",
+  "item.endersdelight.endermite_stew": "末影炖肉",
+  "item.endersdelight.endermite_stew_wood": "末影炖肉",
+  "item.endersdelight.mite_crust": "末影螨壳",
+  "item.endersdelight.pearl_pasta": "珍珠面",
+  "item.endersdelight.pearl_pasta_wood": "珍珠面",
+  "item.endersdelight.shulker_bowl": "潜影贝碗",
+  "item.endersdelight.shulker_filet": "潜影贝菲力",
+  "item.endersdelight.shulker_mollusk": "潜影贝肉",
+  "item.endersdelight.sight_fragment": "眼球碎片",
+  "item.endersdelight.strange_eclair": "奇怪的油酥饼",
+  "item.endersdelight.stuffed_shulker_bowl": "一碗潜影贝馅",
```

```suggestion
  "item.endersdelight.stuffed_shulker_bowl": "碗装填馅潜影贝",
```
农夫乐事译法
这种方块形态放下来再用餐具取的，一般都是`碗装+原菜名` `盘装+原菜名`